Providers/Profile.swift

```diff
@@ -958,21 +958,16 @@ public class BrowserProfile: Profile {
         private func displayStateForSyncState(syncStatus: SyncStatus, identifier: String? = nil) -> SyncDisplayState {
             switch syncStatus {
             case .Completed:
-                return SyncDisplayState.Good
-            case .NotStarted(let reason):
-                let message: String
-                switch reason {
-                case .Offline:
-                    message = Strings.FirefoxSyncOfflineTitle
-                default:
-                    message = Strings.FirefoxSyncNotStartedTitle
-                }
-                return SyncDisplayState.Stale(message: message)
+                return .Good
```

Comment:
If the default case is to return .Good. Should we handle the happy case here explicitly? If not we could remove the .Complete case.